action.yml

```diff
@@ -308,16 +314,21 @@ runs:
 
     - name: PR Comment
       id: pr-comment
+      if: steps.lighthouse.outputs.scores
       uses: actions/github-script@v5
       with:
         github-token: ${{ inputs.GITHUB_BOT_TOKEN || github.token }}
         script: |
-          ("${{ steps.lighthouse.outputs.scores }}" != '') && ("${{ inputs.LHCI_GITHUB_APP_TOKEN }}" == '') && github.rest.issues.createComment({
-            issue_number: ${{ steps.variables.outputs.pullRequestID || 0 }},
-            owner: context.repo.owner,
-            repo: context.repo.repo,
-            body: "Lighthouse <a target='_blank' rel='noopener' href='https://snapui.searchspring.io/${{ steps.variables.outputs.siteId }}/.lighthouse/${{ steps.variables.outputs.branch }}/" + ${{ steps.lighthouse.outputs.report || '""' }}.split('/').pop() + "'>Results</a>\n\n${{ steps.lighthouse.outputs.scores }}".split(',').join('\n')
-          })
+          if (!${{ inputs.testing }}) {
```

Comment:
Can't test this for sure until the syntax error is resolved however this may need to change to
```
if (!"${{ inputs.testing }}") {
```

Reply:
This logic seems to work fine - as the PR used for testing now has the lighthouse comment.